parsec/core/gui/workspace_button.py

```diff
@@ -49,7 +51,13 @@ class WorkspaceButton(QWidget, Ui_WorkspaceButton):
     open_clicked = pyqtSignal(WorkspaceFS)
     switch_clicked = pyqtSignal(bool, WorkspaceFS, object)
 
-    def __init__(self, core, jobs_ctx, workspace_fs, parent=None):
+    def __init__(
+        self,
+        core: LoggedCore,
+        jobs_ctx: Any,
```

Comment:
`Any` does not seems approriate here.
It's seems that its a well defined object `QtToTrioJobSchedule`